lib/src/presentation/widget/display_movie_detail.dart

```diff
@@ -1,28 +1,34 @@
 import 'package:flutter/material.dart';
 import '../../core/util/constants.dart';
 import '../../domain/model/movie.dart';
+import '../../domain/usecase/implementation/genres_usecase_impl.dart';
+import '../bloc/genres_in_movie_detail_bloc.dart';
 import '../view/movie_detail.dart';
 import 'image.dart';
 
 class DisplayMovie extends StatelessWidget {
-  const DisplayMovie({super.key, required this.movie});
+  const DisplayMovie({
+    super.key,
+    required this.movie,
+  });
 
   final Movie movie;
   final SizedBox spaceBetweenData = const SizedBox(
     height: 10,
   );
   final TextStyle movieTitleTextStyle = const TextStyle(
-    fontSize: 24,
+    fontSize: 23,
```

Comment:
Add fontSize value to const

Reply:
if I create a const called fontSize I couldn't use it into the initalization of the movieTitleTextStyle